projects/gloo/cli/pkg/common/get.go

```diff
@@ -128,6 +135,15 @@ func GetProxies(name string, opts *options.Options) (gloov1.ProxyList, error) {
 	}
 	return getProxiesFromK8s(name, opts)
 }
+
+func GetProxiesFromSettings(name string, opts *options.Options, settings *gloov1.Settings) (gloov1.ProxyList, error) {
```

Comment:
Love this! Since it's exported, can be add a docstring

Reply:
```go
// Retrieve proxies from the proxy debug endpoint, or from kubernetes if the proxy debug endpoint is not available
// Takes in a settings object to determine whether the proxy debug endpoint is available
func GetProxiesFromSettings(name string, opts *options.Options, settings *gloov1.Settings) (gloov1.ProxyList, error) {
```

Reply:
Looking good! Can we use the recommendation from https://go.dev/blog/godoc? 

```suggestion go
// GetProxiesFromSettings retrieves proxies from the proxy debug endpoint, or from kubernetes if the proxy debug endpoint is not available
// Takes in a settings object to determine whether the proxy debug endpoint is available
func GetProxiesFromSettings(name string, opts *options.Options, settings *gloov1.Settings) (gloov1.ProxyList, error) {
```

Reply:
Updated here: a1190b83cdc96db9bfe56e1fa132c18c128183e5